src/scheduler/Analysis.py

```diff
@@ -352,6 +360,8 @@ def _handle_analysis_tags(self, fw, plugin):
     def check_further_process_or_complete(self, fw_object):
         if not fw_object.scheduled_analysis:
             logging.info('Analysis Completed:\n{}'.format(fw_object))
+            if isinstance(fw_object, FileObject):
```

Comment:
What else can this be? Given that it has to have the `scheduled_analysis` attribute by now

Reply:
the `Firmware` objects are also analyzed and also have the attribute, right?